spec/fstab_spec.rb

```diff
@@ -1,18 +1,14 @@
 require 'stringio'
 
 describe LinuxAdmin::FSTab do
-  before do
-    # Reset the singleton so subsequent tests get a new instance
-    Singleton.send :__init__, LinuxAdmin::FSTab
-  end
-
   it "has newline, single spaces, tab" do
     fstab = <<eos
 
   
 	
 eos
-    expect(File).to receive(:read).with('/etc/fstab').and_return(fstab)
+    expect(File).to receive(:read).at_least(:once).with('/etc/fstab').and_return(fstab)
```

Comment:
That was to account for if it was the first time we referenced instance and also when we call refresh